sites/all/patches/opensearch-autocomplete.patch

```diff
@@ -55,33 +41,20 @@ index ff141511..0cbe9a3c 100644
 -    );
 -    return $items;
 -  }
--  
--  $url = url($rest_url, array('query' => $query));
 +  // Hack to use old suggestions service.
 +  $query += variable_get('ting_search_autocomplete_settings', array());
 +  $query['query'] = $query['q'];
 +  $query['maxSuggestions'] = $query['rows'];
 +
 +  $method = variable_get('ting_search_autocomplete_method', 'facets');
-+
+ 
+-  $url = url($rest_url, array('query' => $query));
 +  $url = url('http://opensuggestion.addi.dk/b3.5_2.0/rest/'.$method, array('query' => $query));
```

Comment:
Should use the URL set in the admin and store in: `variable_get('opensearch_search_autocomplete_suggestion_url', '');`